modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -44,12 +44,25 @@ public class UpgradeCatchAllCheck {
 			companyId, 551, "test", start, end);
 	}
 
+	public void method(
+		long userId, long groupId, long parentFolderId, String name,
+		String description, ServiceContext serviceContext) {
+
+		_journalFolderLocalService.addFolder(null, userId, groupId, parentFolderId, name, description, serviceContext);
+
```

Comment:
@alberttgomes please add one case of success for the Util class.

Reply:
Let me ask a question. The pattern to follow is always a case of success for Util and Service, and a case of wrong for Util and Service, right?